command/genesis/polybft_params.go

```diff
@@ -134,24 +140,71 @@ func (p *genesisParams) generatePolyBftChainConfig(o command.OutputFormatter) er
 		}
 	}
 
+	voteDelay, err := common.ParseUint256orHex(&p.voteDelay)
+	if err != nil {
+		return err
+	}
+
+	votingPeriod, err := common.ParseUint256orHex(&p.votingPeriod)
+	if err != nil {
+		return err
+	}
+
+	if votingPeriod.Cmp(big.NewInt(0)) == 0 {
+		return errInvalidVotingPeriod
+	}
+
+	proposalThreshold, err := common.ParseUint256orHex(&p.proposalThreshold)
+	if err != nil {
+		return err
+	}
+
+	governorAdminAddr := types.ZeroAddress
+	// if no admin is defined, zero address will be the owner,
+	// meaning no new proposers and executors besides genesis validator
+	// set can be added/removed later
+	if p.governorAdmin != "" {
+		governorAdminAddr = types.StringToAddress(p.governorAdmin)
+	}
+
+	proposalQuorum := p.proposalQuorum
+	if proposalQuorum > 100 {
+		// proposal can be from 0 to 100, so we sanitize the value
+		proposalQuorum = 100
```

Comment:
Let's move 100 to constants or as a default value of `p.proposalQuorum` per its definition in flags

Reply:
It's default is 67% (2/3 + 1) like we do in consensus quorum, so I wouldn't set it by default to 100%, but I can move this 100% to a constant.

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/1519/commits/4a92ab07bf4ef242b2a664bf3fa2c71509a7289d